### ISSUE TITLE
dependencies require manual install

### ISSUE THREAD

**Issue by tvogt** [grand-stack/grand-stack-starter#7]
```
npm WARN apollo-cache-control@0.2.0-rc.1 requires a peer of graphql@0.10.x - 0.13.x but none is installed. You must install peer dependencies yourself.
npm WARN graphql-extensions@0.1.0-beta.0 requires a peer of graphql@0.10.x - 0.13.x but none is installed. You must install peer dependencies yourself.
npm WARN apollo-cache-inmemory@1.2.5 requires a peer of graphql@0.11.7 || ^0.12.0 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-client@2.3.5 requires a peer of graphql@^0.11.0 || ^0.12.0 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-link@1.2.2 requires a peer of graphql@^0.11.3 || ^0.12.3 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-link-http@1.5.4 requires a peer of graphql@^0.11.0 || ^0.12.0 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-link-http-common@0.2.4 requires a peer of graphql@^0.11.0 || ^0.12.0 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-server@2.0.0-rc.5 requires a peer of graphql@^0.12.0 || ^0.13.0 || ^14.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-server-core@2.0.0-rc.8 requires a peer of graphql@^0.12.0 || ^0.13.0 || ^14.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-server-errors@2.0.0-rc.8 requires a peer of graphql@^0.12.0 || ^0.13.0 || ^14.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-server-express@2.0.0-rc.8 requires a peer of graphql@^0.12.0 || ^0.13.0 || ^14.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-tracing@0.2.0-rc.1 requires a peer of graphql@0.10.x - 0.13.x but none is installed. You must install peer dependencies yourself.
npm WARN graphql-extensions@0.1.0-beta.0 requires a peer of graphql@0.10.x - 0.13.x but none is installed. You must install peer dependencies yourself.
npm WARN apollo-upload-server@5.0.0 requires a peer of graphql@^0.13.1 but none is installed. You must install peer dependencies yourself.
npm WARN graphql-extensions@0.1.0-rc.2 requires a peer of graphql@0.10.x - 0.13.x but none is installed. You must install peer dependencies yourself.
npm WARN graphql-subscriptions@0.5.8 requires a peer of graphql@^0.10.5 || ^0.11.3 || ^0.12.0 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN graphql-tag@2.9.2 requires a peer of graphql@^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN graphql-tools@3.0.5 requires a peer of graphql@^0.12.0 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN subscriptions-transport-ws@0.9.12 requires a peer of graphql@^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.1 but none is installed. You must install peer dependencies yourself.
npm WARN grand-stack-starter-api@0.0.1 No repository field.

```

**Comment by tvogt** [comment]
this is on the api. the ui installed clear. git clone was done today.

**Comment by tvogt** [comment]
manual `npm install graphq`l solves this, so the graphql dependency is to blame

**Comment by tvogt** [comment]
after this it fails to start:

```
$ npm start

> grand-stack-starter-api@0.0.1 start /Users/tom/Code/grand-stack-starter/api
> nodemon --exec babel-node src/index.js

[nodemon] 1.18.2
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `babel-node src/index.js`
/Users/tom/Code/grand-stack-starter/api/node_modules/neo4j-graphql-js/node_modules/graphql/jsutils/instanceOf.js:37
      throw new Error("Cannot use ".concat(className, " \"").concat(value, "\" from another module or realm.\n\nEnsure that there is only one instance of \"graphql\" in the node_modules\ndirectory. If different versions of \"graphql\" are the dependencies of other\nrelied on modules, use \"resolutions\" to ensure only one version is installed.\n\nhttps://yarnpkg.com/en/docs/selective-version-resolutions\n\nDuplicate \"graphql\" modules cannot be used at the same time since different\nversions may have different capabilities and behavior. The data from one\nversion used in the function from another could produce confusing and\nspurious results."));
      ^

Error: Cannot use GraphQLObjectType "__Directive" from another module or realm.

Ensure that there is only one instance of "graphql" in the node_modules
directory. If different versions of "graphql" are the dependencies of other
relied on modules, use "resolutions" to ensure only one version is installed.

https://yarnpkg.com/en/docs/selective-version-resolutions

Duplicate "graphql" modules cannot be used at the same time since different
versions may have different capabilities and behavior. The data from one
version used in the function from another could produce confusing and
spurious results.
    at instanceOf (/Users/tom/Code/grand-stack-starter/api/node_modules/neo4j-graphql-js/node_modules/graphql/jsutils/instanceOf.js:37:13)
    at isObjectType (/Users/tom/Code/grand-stack-starter/api/node_modules/neo4j-graphql-js/node_modules/graphql/type/definition.js:93:34)
    at isNamedType (/Users/tom/Code/grand-stack-starter/api/node_modules/neo4j-graphql-js/node_modules/graphql/type/definition.js:338:32)
    at isSpecifiedScalarType (/Users/tom/Code/grand-stack-starter/api/node_modules/neo4j-graphql-js/node_modules/graphql/type/scalars.js:156:38)
    at isDefinedType (/Users/tom/Code/grand-stack-starter/api/node_modules/neo4j-graphql-js/node_modules/graphql/utilities/schemaPrinter.js:57:46)
    at Array.filter (<anonymous>)
    at printFilteredSchema (/Users/tom/Code/grand-stack-starter/api/node_modules/neo4j-graphql-js/node_modules/graphql/utilities/schemaPrinter.js:65:6)
    at printSchema (/Users/tom/Code/grand-stack-starter/api/node_modules/neo4j-graphql-js/node_modules/graphql/utilities/schemaPrinter.js:47:10)
    at addMutationsToSchema (/Users/tom/Code/grand-stack-starter/api/node_modules/neo4j-graphql-js/dist/augmentSchema.js:28:52)
    at augmentSchema (/Users/tom/Code/grand-stack-starter/api/node_modules/neo4j-graphql-js/dist/index.js:320:64)
[nodemon] app crashed - waiting for file changes before starting...


```

**Comment by johnymontana** [comment]
This should be fixed in 29fb5d88576be6c0c8215d80c6e319b000373f48

The problem was that one of the dependencies of grand-stack-starter, neo4j-graphql-js, was previously using a release candidate version of graphql-js which did not resolve the peer dependency requirements of the other packages. neo4j-graphql-js now uses the latest stable release of graphql-js, which should fix this problem.

Closing the issue, but please reopen if you experience the problem again.

**Comment by po** [comment]
Why am I getting warnings if the ticket is closed yet the dependencies still need to be updated?
```
npm install graphql-yoga                              ✓  11  23:11:17
npm WARN deprecated apollo-upload-server@5.0.0: Please migrate to https://npm.im/graphql-upload (see https://git.io/fADhl).
npm WARN apollo-upload-server@5.0.0 requires a peer of graphql@^0.13.1 but none is installed. You must install peer dependencies yourself.
npm WARN graphql-subscriptions@0.5.8 requires a peer of graphql@^0.10.5 || ^0.11.3 || ^0.12.0 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN graphql-middleware@1.7.7 requires a peer of graphql@^0.13.2 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-server-core@1.4.0 requires a peer of graphql@^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-tracing@0.1.4 requires a peer of graphql@0.10.x - 0.13.x but none is installed. You must install peer dependencies yourself.
npm WARN apollo-cache-control@0.1.1 requires a peer of graphql@0.10.x - 0.13.x but none is installed. Youmust install peer dependencies yourself.
npm WARN graphql-extensions@0.0.10 requires a peer of graphql@0.10.x - 0.13.x but none is installed. You must install peer dependencies yourself.
npm WARN mern@1.0.0 No repository field.```
